File: src/Ejercicio2/Ejercicio2.java
package Ejercicio2;

import java.util.Scanner;

public class Ejercicio2 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        try {
            System.out.print("Ingrese un número: ");
            int n = Integer.parseInt(sc.nextLine());

            if (n%2 == 0) System.out.println("El número " + n + "es par.");
            else System.out.println("El número " + n + "es impar.");
        } catch (Exception e) {
            System.out.println("El valor ingresado no era un número");
        }
    }
}

File: src/Ejercicio2/Ejercicio2.kt
package Ejercicio2

fun main() {

    try{
        print("Ingrese un número: ")
        val num = readln().toInt()
        if (num % 2 == 0) println("El numero $num es par")
        else println("El numero $num es impar")
    }
    catch (e: Exception){
        println("El valor ingresado no era un número")
    }

}